tests/test_rules.py

```diff
@@ -126,31 +127,42 @@ def test_duplicate_tags(self):
                          "There are rules with duplicate tags")
 
     def test_look_for_duplicate_filters(self):
-        def check_list_or_recurse_on_dict(item, depth: int) -> None:
+        def check_list_or_recurse_on_dict(item, depth: int, special: int) -> None:
             if type(item) == list:
-                check_if_list_contain_duplicates(item, depth)
+                check_if_list_contain_duplicates(item, depth, special)
             elif type(item) == dict and depth <= MAX_DEPTH:
-                for sub_item in item.values():
-                    check_list_or_recurse_on_dict(sub_item, depth + 1)
+                for keys, sub_item in item.items():
+                    if "|base64offset" in keys:
```

Comment:
`|re` should be case-sensitive as well

Reply:
`|base64` exists also

Reply:
I'll add `base64` but the `re` no need because its a single item list and you need a whole different test to find duplicates between different lists